src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Backup and Restore
+description: Backup and Restore your Okteto instance
+id: backup
+sidebar_label: Backup and Restore
+---
+
+Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
+This guide outlines the steps to recover an Okteto instance in such scenarios.
+
+## Critical Resources
+
+To effectively recover an Okteto instance, focus on the following critical resources:
+
+- **Service Accounts**: Preserve service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
+- **Secrets**: Recreate all secrets within the `okteto` namespace.
+- **Custom Resources**: Restore Custom Resources (CRs) corresponding to Okteto's CRDs. They can be identified with the label `app.kubernetes.io/part-of: okteto`.
+
+## Recovery Process
+
+You can follow two different recovery processes:
+
+- **Light recovery processes**: recovers all the users’ metadata, tokens and secrets, but it doesn’t recover users’ namespaces and volumes. All this information is available on the `okteto` namespace.
+- **Full recovery processes**: recovers not only the `okteto` namespace, but also all the users’ namespaces and preview environments, including data in persistent volumes.
+
+### Light Recovery Process
+
+If you do not wish to retain users’ namespaces and their application data, migrating the resources in the `okteto` namespace is sufficient. This is the fastest option, but your users must recreate their development environments.
+
+Follow these steps:
+
+- **Backup the okteto namespace**: backup the [critical resources](self-hosted/administration/configuration.mdx#subdomain) in your `okteto` namespace.
```

Comment:
```suggestion
- **Backup the okteto namespace**: backup the [critical resources](self-hosted/administration/configuration.mdx#subdomain) in your `okteto` namespace.
```

This link on `critical resources` seems odd. Is this intentional and if so, why is the `#subdomain` anchor content critical?

Reply:
I read that docs page and I think I have a better idea of why you're linking here. Is this ultimately about "backing up" the DNS entry for the desired subdomain used for the namespace? If so, we might just explain that here since it's relatively short/small content rather than link to a section on a bigger page that doesn't "get right to the point" since it's serving a different purpose elsewhere.

WDYT?

Reply:
It's a typo, I wanted to link to the previous section in this doc. Instead of "Critical resources" I am thinking to rename it to "Okteto Metadata", or "Okteto internal data"